### ISSUE TITLE
could you please add a keyboard shortcut to take, or add ...

### ISSUE THREAD

**Issue by Aluxian** [Aluxian/Whatsie#113]
**Description:**
could you please add a keyboard shortcut to take, or add a picture?  The primary reason I like this app is that it allows me to communicate through WhatsApp more efficiently, just as how WhatsApp web allows me to communicate more efficiently using a full size keyboard.  One thing that a dedicated desktop app can do, is extend keyboard shortcuts (and other features) in ways impossible or contrary to the design philosophy of a web browser, such as ctrl+o to open an image to attach, and ctrl+1, ctrl+2, etc for your favorite chats, or tabs for your favorite chats.  
You can also add functionality currently missing in the webapp, such as up/down to move between chats when the textarea is empty.  
Your app is already great, but with enough compelling features it could become standard, even a household name, such as how some of Rudy Huyn's apps have displaced some native Windows Phone apps, but there is much more glory in developing for Windows/Mac/Debian than for Windows Phone (though that is not a bad platform).

**Best thing about Whatsie:**
Ease of use, Design,

**Worst thing about Whatsie:**
lack of keyboard shortcuts

**Extra:**

**Comment by Aluxian** [comment]
Keyboard shortcuts for:
- [ ] take photo
- [ ] add/send photo (ctrl+o?)
- [ ]

**Comment by LawranceFung** [comment]
yes, but up/down also switches between chats if the message box is empty

**Comment by Aluxian** [comment]
@LawranceFung There's Ctrl+Up and Ctrl+Down (Cmd+Up and Cmd+Down on OS X) already for switching conversations.

**Comment by LawranceFung** [comment]
Would it be possible to add that to the menubar?  It seems that that is where all the keyboard shortcuts are documented to the user.

**Comment by Aluxian** [comment]
That's a good idea. I will, thanks!
